tracer/test/test-applications/integrations/Samples.Kafka/Producer.cs

```diff
@@ -60,7 +66,13 @@ public static void Produce(string topic, int numMessages, ClientConfig config, b
 
         private static void Produce(string topic, int numMessages, ClientConfig config, Action<DeliveryReport<string, string>> deliveryHandler, bool isTombstone)
         {
-            using (var producer = new ProducerBuilder<string, string>(config).Build())
+            var producerConfig = new ProducerConfig
+            {
+                BootstrapServers = config.BootstrapServers,
+                MessageTimeoutMs = 3000 // earlier versions would return right away when producing to invalid topics - later versions would block for 30 seconds
+            };
+
+            using (var producer = new ProducerBuilder<string, string>(producerConfig).Build())
```

Comment:
On later versions of the Kafka NuGet (1.9.3/2.2.0) the tests were taking an additional 60 seconds each. This was when we produce to an invalid topic each time on these later versions they'd wait for 30 seconds where on the older versions they'd just immediately return.

I'm not super familiar with Kafka, but searching through their release logs there were a few changes of increasing timeouts/retries and this seems to bring the times back down.

**But - the messages timeout now on the later versions instead of failing for "Invalid topic"**
The tests still pass with this though

Saved ~4 minutes per integration test run